### DISCUSSION TITLE
Looking for a tutorial

### DISCUSSION THREAD

**Discussion by costantino78** [isce-framework/fringe#80] [Q&A] [answered]
Hi all.
Looking for a free library that implements an algorithm SqueeSAR-like I discovered FRiNGE. I understand that FRiNGE uses both PS and DS.

I have two questions:

1. Does FRiNGE produce a time series or is it a preprocessor, so the result must be ingested to a tool like MintPY to produce the displacement time series?
2. Is there a detailed jupyter notebook that explains the entire process from acquiring Sentinel-1 data to developing the displacement time series?

Regards
Costantino

**Comment by scottstanie** [comment, marked as answer]
Hi Costantino,

1. Yes fringe is a "preprocessor" in that the furthest along [the workflow here](https://github.com/isce-framework/fringe/blob/main/docs/workflows.md#phase-2-wrapped-time-series-to-deformation-time-series) gets is to unwrap the interferograms produced by the phase linking. If you wanted to apply corrections, invert a network that has more than the single-reference interferograms, etc., you would need something like Mintpy. A script has been [added here](https://github.com/insarlab/MintPy/blob/main/src/mintpy/prep_fringe.py) that prepares the fringe outputs for Mintpy.

2. The most detailed notebooks I know for general acquistion-to-displacement-time-series are from the [UNAVCO short course](https://github.com/parosen/Geo-SInC/tree/main/UNAVCO2022), which uses the isce2 processor to get stacks of unwrapped interferograms, then feeds them to mintpy for time series analysis. While the algorithms in fringe are mentioned in one of the lecture, there is not a detailed notebook for using fringe there.

While the developers of Fringe are unlikely to create new detailed learning material for using Fringe, there may be notebooks in the future for https://github.com/opera-adt/dolphin which the OPERA project is currently working on for large scale processing using a DS/PS algorithm.

**Comment by costantino78** [comment]
Thanks for the replies, they are very helpful!

Costantino
